tensornetwork/tn_keras/mpo.py

```diff
@@ -0,0 +1,167 @@
+import tensorflow as tf
+from tensorflow.keras.layers import Layer  # type: ignore
+from tensorflow.keras import activations
+from tensorflow.keras import initializers
+from typing import List, Optional, Text, Tuple
+import tensornetwork as tn
+from tensornetwork.network_components import Node
+import numpy as np
+import math
+
+
+class DenseMPO(Layer):
```

Comment:
This layer needs a get_config() method.

Reply:
Sounds great - added `get_config` to both layers.